src/ui/units/connections/components/custom-forms/Yadocs/components/AdditionalTitleContent.tsx

```diff
@@ -3,19 +3,39 @@ import React from 'react';
 import {HelpPopover} from '@gravity-ui/components';
 import {Button, Checkbox, Icon} from '@gravity-ui/uikit';
 import block from 'bem-cn-lite';
+import {ConnectorType} from 'shared';
+import {registry} from 'ui/registry';
 
+import {ButtonLogout} from '../../components';
 import {i18n8857} from '../constants';
 
 import iconSync from '../../../../../../assets/icons/sync.svg';
 
 const b = block('conn-form-yadocs');
 const ICON_SIZE = 18;
 
-export const AdditionalTitleContent = () => {
+type Props = {
+    authorized: boolean;
+    clickLoginButton: (oauthToken: string) => void;
```

Comment:
maybe **on**ClickLoginButton?

Reply:
Such naming is already [here](https://github.com/datalens-tech/datalens-ui/blob/4c0dabd8c115be056b81dc52b39aceffd4b51aae/src/ui/units/connections/components/custom-forms/Yadocs/containers/DocsListContainer.tsx#L50). Plus it's a transit option, I'm not sure it's worth calling it that.

Reply:
It seems to me that it is logical for internal handlers and for handlers for events coming from outside (in props) to have different prefixes

I'm expecting an onEvent in props and something like a handleEvent inside the component.

Reply:
What about props that drills into a deep?